modules/appeals_api/app/controllers/concerns/appeals_api/openid_auth.rb

```diff
@@ -64,7 +64,7 @@ def token_validation_result
     private
 
     def unsafe_mode?
-      Rails.env.development? && Settings.dig(:modules_appeals_api, :enable_unsafe_mode)
+      Rails.env.development? && Settings.modules_appeals_api.enable_unsafe_mode
```

Comment:
@department-of-veterans-affairs/lighthouse-banana-peels This setting doesn't actually exist anywhere... It's not in settings.yml or in any of the values.yaml in the manifest repo.